CMakeLists.txt

```diff
@@ -183,6 +198,7 @@ set(XEUS_CPP_HEADERS
     include/xeus-cpp/xmagics.hpp
     include/xeus-cpp/xoptions.hpp
     include/xeus-cpp/xpreamble.hpp
+    include/xeus-cpp/xeus_cpp_tags_config.hpp
```

Comment:
The name should be more generic. xeus_cpp_config is probably what we need.

Reply:
That's what I tried previously and the file was overridden. Had a conversation about this with @mcbarton and the way to go about saving the definitions was creating a new file.

Reply:
We should merge both files into one and it will work. Note that we are writing to `xeus_cpp_config` and that should probably change, too.

Reply:
@vgvassilev I have helped @tharun571 to combine the config files and it is ready for another review